### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by MehdiChinoune**

### CODE REVIEW

File: web/docs/ci.md [hunk 1]
```diff
@@ -63,6 +63,33 @@ $env:MSYSTEM = 'MINGW64'  # Start a 64 bit Mingw environment
...
+  script:
+    - Invoke-WebRequest -Uri 'https://github.com/msys2/msys2-installer/releases/download/nightly-x86_64/msys2-base-x86_64-latest.sfx.exe' -OutFile 'msys2.exe'
+    - .\msys2.exe -y -oC:\
+    - Remove-Item msys2.exe
+    - $env:PATH = "C:\msys64\ucrt64\bin;C:\msys64\usr\bin;$env:PATH"
+    - bash -lc ' '
+    - pacman --noconfirm -Syuu
+    - pacman --noconfirm -Syuu
+    - $env:CHERE_INVOKING = 'yes'
+    - $env:MSYSTEM = 'MINW64'
```
Comment (lazka) [inline web/docs/ci.md:89]:
```suggestion
    - $env:MSYSTEM = 'MINGW64'
```

File: web/docs/ci.md [hunk 2]
```diff
@@ -63,6 +63,33 @@ $env:MSYSTEM = 'MINGW64'  # Start a 64 bit Mingw environment
...
+  stage: build_mingw
+  tags:
+    - shared-windows
+    - windows
+    - windows-1809
+  script:
+    - Invoke-WebRequest -Uri 'https://github.com/msys2/msys2-installer/releases/download/nightly-x86_64/msys2-base-x86_64-latest.sfx.exe' -OutFile 'msys2.exe'
+    - .\msys2.exe -y -oC:\
+    - Remove-Item msys2.exe
+    - $env:PATH = "C:\msys64\ucrt64\bin;C:\msys64\usr\bin;$env:PATH"
```
Comment (lazka) [inline web/docs/ci.md:84]:
```suggestion
    - $env:PATH = "C:\msys64\mingw64\bin;C:\msys64\usr\bin;$env:PATH"
```

**Comment by lazka** [comment]
Any reason you are not doing the same thing as other examples with a login shell?

**Comment by MehdiChinoune** [comment]
> Any reason you are not doing the same thing as other examples with a login shell? Just wondering...

What do you mean by login shell,
do you mean defining msys as `bash -lc ' '` and use to execute other commands?

**Comment by MehdiChinoune** [comment]
> I mean running everything in a login shell, and also run the build script there, so you don't have to set PATH manually, like here: https://www.msys2.org/docs/ci/#other-systems

I just didn't think of that!
I think the example should be updated and tested.

**Comment by MehdiChinoune** [comment]
> I mean running everything in a login shell, and also run the build script there, so you don't have to set PATH manually, like here: https://www.msys2.org/docs/ci/#other-systems
> 
> Not that there is much difference, but imo having things consistent on the same page is less confusing.

I think defining PATH for once and the executing commands is much better, at least for MINGW environments.

**Comment by lazka** [comment]
hm?

**Pull request closed without merging**